src/components/purchase-complete/index.js

```diff
@@ -62,12 +62,46 @@ const PurchaseComplete = ({
 
     if (!checkout) return null;
 
-    let footerHasTicketText = (rest.hasOwnProperty('footerHasTicketText') && typeof rest.footerHasTicketText !== 'undefined' ?
-        rest.footerHasTicketText : T.translate('purchase_complete_step.footer_has_ticket_text'));
+    let orderComplete1stParagraph = (
+        currentUserTicket ? 
+            rest.hasOwnProperty('orderComplete1stParagraph') && typeof rest.orderComplete1stParagraph !== 'undefined' ?
+                rest.orderComplete1stParagraph
+                :
+                T.translate('purchase_complete_step.finish_now_label')
+            : 
+            rest.hasOwnProperty('initialOrderComplete1stParagraph') && typeof rest.initialOrderComplete1stParagraph !== 'undefined' ?
+                rest.initialOrderComplete1stParagraph
+                : 
+                T.translate('purchase_complete_step.my_orders_label')
+    );
+
+    let orderCompleteButtonText = (
+        currentUserTicket ? 
+            rest.hasOwnProperty('orderCompleteButton') && typeof rest.orderCompleteButton !== 'undefined' ?
```

Comment:
if currentUserTicket == true , then its means that its the initial order, so, should be using here
if defined , initialOrderCompleteButton otherwise null